var/spack/repos/builtin/packages/cnvnator/package.py

```diff
@@ -0,0 +1,67 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Cnvnator(MakefilePackage):
+    """A tool for CNV discovery and genotyping
+    from depth-of-coverage by mapped reads."""
+
+    homepage = "https://github.com/abyzovlab/CNVnator"
+    url      = "https://github.com/abyzovlab/CNVnator/archive/v0.3.3.tar.gz"
+
+    version('0.3.3', 'f9fc0c2e79abe85decab00d3748d8904')
+
+    depends_on('samtools')
+    depends_on('htslib')
+    depends_on('root')
+    depends_on('bzip2')
+    depends_on('curl')
+    depends_on('lzma')
+    depends_on('zlib')
+
+    def edit(self, spec, prefix):
+        makefile = FileFilter('Makefile')
+        # Replace CXX with CXXFLAGS
+        makefile.filter('CXX.*=.*',
+                        'CXXFLAGS = -O3 -std=c++11 \
+                        -DCNVNATOR_VERSION=\\"$(VERSION)\\" $(OMPFLAGS)')
```

Comment:
Will this string contain a ton of spaces from the indentation? I think you can add closing and starting quotes and Python will know to combine it into a single string. Also, adding a `r` before the string makes it raw, and means that you no longer need to escape backslashes.

Reply:
Also, the `-fopenmp` flag used in Makefile is compiler specific. You should replace it with `self.compiler.openmp_flag`.